arches_lingo/src/arches_lingo/components/basic-search/SearchResult.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import type { Concept } from "@/arches_lingo/types.ts";
+import { getItemLabel } from "@/arches_lingo/utils.ts";
+
+defineProps({
+    searchResult: {
+        type: Object,
```

Comment:
if there's no type we can import from PrimeVue for this, we could just explode this out into two props.